src/desktop/analytics/account_creation.js

```diff
@@ -35,12 +39,12 @@ const trackAccountCreation = options => {
 }
 
 // Created account (via email)
-$(document).on(
+$(document).one(
   'submit',
   '.auth-register form, .marketing-signup-modal form, .artist-page-cta-overlay__register form, .gdpr-signup form',
-  function() {
+  () => {
     $(document).one('ajaxComplete', (e, xhr, options) =>
-      trackAccountCreation({
+      mediator.trigger('auth:sign_up:email', {
```

Comment:
Since that's essentially updating how we track account creations for any sign ups forms not just for tooltips, does this support the new analytics schema we settled on?

Reply:
Yeah, I've extended the existing schema with the new terms we are using here: https://github.com/artsy/force/pull/2571/files#diff-61672d4a0f7a6e21d06cfdfa3a5efe85R24

Reply:
Awesome, we're good then